### PR TITLE
chore: account for unverified profiles

### PR DESCRIPTION
**Pull request by thelostone-mc**
Screenshot 2020-06-16 at 11 02 36 PM

### CODE REVIEW

File: app/grants/clr.py [hunk 1]
```diff
@@ -95,8 +102,8 @@ def translate_data(grants_data):
 '''
 def get_verified_list(grant_contributions):
     verified_list = []
-    for proj, user, ver_stat, amount in grant_contributions:
-        if ver_stat == 'verified' and user not in verified_list:
+    for _, user, ver_stat, _ in grant_contributions:
+        if ver_stat and user not in verified_list:
```
Comment (frankchen07) [inline app/grants/clr.py:106]: hm, why do we need if `ver_stat` not in `verified_list` if the list is primarily users?

File: app/grants/clr.py [hunk 2]
```diff
@@ -346,30 +354,40 @@ def fetch_data(clr_type=None, network='mainnet', clr_start_date=None, clr_end_da
...
     elif clr_type == 'matic':
         grants = Grant.objects.filter(network=network, hidden=False, active=True, grant_type='matic', link_to_new_grant=None)
-        threshold = THRESHOLD_MATIC
+        v_threshold = V_THRESHOLD_MATIC
+        uv_threshold = UV_THRESHOLD_MATIC
         total_pot = TOTAL_POT_MATIC
+
     else:
-        return None, None, None, None
+        return None, None, None, None, None, None
```
Comment (frankchen07) [inline app/grants/clr.py:386]: is the purpose of this to not have it error out if its empty?

**Comment by frankchen07** [comment]
left some comments / questions that won't affect merging. last change would be to change uv_threshold to the estimate that would give a 20% difference between verified and non verified

File: app/grants/clr.py (hunk 2 above)
Comment (thelostone-mc) [reply app/grants/clr.py:386]: Yup! Just being safe

File: app/grants/clr.py (hunk 1 above)
Comment (thelostone-mc) [reply app/grants/clr.py:106]: I'm not clear here ! Isn't that what you intended when you wrote it up?

Comment (frankchen07) [reply app/grants/clr.py:106]: sorry wasn't thinking right on the ver_stat variable. it's boolean now, not a string. 👍

**Pull request merged**